README.md

```diff
@@ -17,10 +17,15 @@
 
 # Apache Uniffle (Incubating)
 
-Uniffle is a unified remote shuffle service for distributed compute engines.
-It provides the ability to aggregate and store shuffle data on remote servers,
-thus improving the performance and reliability of large jobs.
-Currently it supports [Apache Spark](https://spark.apache.org) and [Apache Hadoop MapReduce](https://hadoop.apache.org).
+Uniffle is a high performance, general purpose remote shuffle service for distributed compute engines.
```

Comment:
There is much more search result "compute engine" than "computing engine":
(but probably it's because GCP's product is called "compute engine")
<img width="305" alt="image" src="https://user-images.githubusercontent.com/5821159/221096511-b4f679ad-e25c-4f7c-9851-66fe9eb1f8d4.png">
<img width="271" alt="image" src="https://user-images.githubusercontent.com/5821159/221096547-0b6ea72e-332c-4a16-bc60-7beb1c11e36a.png">

Reply:
Thanks. More "distributed computing engine" than "distributed compute engine" is found in Google scholar:
<img width="445" alt="image" src="https://user-images.githubusercontent.com/5821159/221096824-47410c43-eabb-4361-a3c9-9f01d16dfee6.png">
<img width="442" alt="image" src="https://user-images.githubusercontent.com/5821159/221096879-4bde5401-05e9-443b-aa1c-feaa90be9f56.png">